adapter.go

```diff
@@ -105,6 +105,9 @@ func (a *Adapter) close() {
 
 // LoadPolicy loads all of policys from ETCD
 func (a *Adapter) LoadPolicy(model model.Model) error {
+	// Emits the time required to load the policy from storage
```

Comment:
We may want to remove this one, put this log in callback if needed with proper logger.

Reply:
Sure. I have added one in the callback. I'll remove this.